ft-sdk/src/auth/provider.rs

```diff
@@ -296,29 +296,25 @@ pub fn create_user(
 /// retrieved without a db call to show a user identifiable information.
 pub fn login(
     conn: &mut ft_sdk::Connection,
-    ft_sdk::UserId(user_id): &ft_sdk::UserId,
-    session_id: Option<ft_sdk::auth::SessionID>,
-) -> Result<ft_sdk::auth::SessionID, LoginError> {
+    user_id: &ft_sdk::UserId,
+    session_id: Option<ft_sdk::session::SessionID>,
+) -> Result<ft_sdk::session::SessionID, LoginError> {
     match session_id {
-        Some(session_id) if session_id.0 == "hello" => {
-            Ok(ft_sdk::auth::session::create_with_user(conn, *user_id)?)
-        }
-        Some(session_id) => Ok(ft_sdk::auth::session::set_user_id(
-            conn, session_id, *user_id,
+        Some(session_id) => Ok(session_id.set_user_id(
+            conn, user_id.clone(),
         )?),
-        None => Ok(ft_sdk::auth::session::create_with_user(conn, *user_id)?),
+        None => Ok(ft_sdk::session::SessionID::new(conn, Some(user_id.clone()), None)?),
```

Comment:
I prefer `create()` over `new()` as conventionally `new()` is used for pure rust construction.

Reply:
done here https://github.com/FifthTry/ft-sdk/pull/52/commits/25f7f56d9c8d472115d553d782b5296e5aaeef03#diff-e8a7a5cb49a5b7d5ea0929541946be6b7bf5be799b2a00093fa352e33bfd6efbL9